conf/modules.config

```diff
@@ -73,7 +73,7 @@ process {
     }
 
     withName: CONVERT_CRAM {
-        ext.args = "--output-fmt cram"
+        ext.args = "--output-fmt cram,archive,level=8"
```

Comment:
I would leave this one as it was. This CRAM file is only there because the chunking requires CRAM. It's purely internal and the file will get deleted during the cleanup process when the pipeline completes.
Therefore, I think it's OK to _not_ use `archive,level=8` here. According to the official benchmark, there's roughly a 10-15% size increase but the conversion is 10-15 times faster !